src/patternfly/components/TreeView/tree-view.scss

```diff
@@ -0,0 +1,196 @@
+  --pf-c-tree-view__list--m-nested-level-1__list-item__link--PaddingLeft: calc(var(--pf-c-tree-view__link--PaddingLeft) + var(--pf-global--spacer--md));
+  --pf-c-tree-view__list--m-nested-level-2__list-item__link--PaddingLeft: calc(var(--pf-c-tree-view__list--m-nested-level-1__list-item__link--PaddingLeft) + var(--pf-global--spacer--lg) + var(--pf-global--spacer--sm));
+  --pf-c-tree-view__list--m-nested-level-3__list-item__link--PaddingLeft: calc(var(--pf-c-tree-view__list--m-nested-level-2__list-item__link--PaddingLeft) + var(--pf-global--spacer--md) + var(--pf-global--spacer--sm));
+  --pf-c-tree-view__list--m-nested-level-1__list-item--m-expandable__link--PaddingLeft: calc(var(--pf-c-tree-view__link--PaddingLeft) + var(--pf-global--spacer--md));
+  --pf-c-tree-view__list--m-nested-level-2__list-item--m-expandable__link--PaddingLeft: calc(var(--pf-c-tree-view__list--m-nested-level-1__list-item--m-expandable__link--PaddingLeft) + var(--pf-global--spacer--md));
+  --pf-c-tree-view__list--m-nested-level-3__list-item--m-expandable__link--PaddingLeft: calc(var(--pf-c-tree-view__list--m-nested-level-2__list-item--m-expandable__link--PaddingLeft) + var(--pf-global--spacer--md));
+
+  // link hover
+  --pf-c-tree-view__link--hover--BackgroundColor: var(--pf-global--BackgroundColor--200);
+
+  // toggle
+  --pf-c-tree-view__link--m-expanded__toggle--Rotate: 90deg;
+  --pf-c-tree-view__link-toggle--PaddingRight: var(--pf-global--spacer--xs);
+  --pf-c-tree-view__link-toggle--PaddingLeft: var(--pf-global--spacer--xs);
+  --pf-c-tree-view__link-toggle--MinWidth: var(--pf-global--spacer--lg);
+  --pf-c-tree-view__link-toggle--MinHeight: var(--pf-c-tree-view__link-toggle--MinWidth);
+
+  // check
+  --pf-c-tree-view__link--c-check--PaddingRight: var(--pf-global--spacer--sm);
+
+  // badge
+  --pf-c-tree-view__link--c-badge--MarginLeft: var(--pf-global--spacer--xs);
+  --pf-c-tree-view__link--c-badge--BackgroundColor: var(--pf-global--disabled-color--200);
+
+  // search
+  --pf-c-tree-view__search--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-tree-view__search--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-tree-view__search--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-tree-view__search--PaddingLeft: var(--pf-global--spacer--sm);
+
+  // icon
+  --pf-c-tree-view__link-icon--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-tree-view__link-icon--Color: var(--pf-global--icon--Color--light);
+
+  // action
+  --pf-c-tree-view__action--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-tree-view__action--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-tree-view__action--Color: var(--pf-global--icon--Color--light);
+  --pf-c-tree-view__action--m-favorite--Color: var(--pf-global--palette--gold-400);
+
+  padding-top: var(--pf-c-tree-view--PaddingTop);
+  padding-bottom: var(--pf-c-tree-view--PaddingBottom);
+}
+
+// stylelint-disable
+.pf-c-tree-view__list {
+  &.pf-m-nested-level-1 {
```

Comment:
Hmmmm.... this section is tricky. Off the top of my head, I think we could do something like this, but I bet there is a better way.

```
--nested-padding: 16px;
.pf-c-tree-view__list {
  $parent = &;
  & & {
    __link--PaddingLeft: --nested-padding;
    & #{$parent} {
      __link--PaddingLeft: calc(--nested-padding * 2);
      & #{$parent} {
        __link--PaddingLeft: calc(--nested-padding * 3);
        // and so on...
      }
    }
  }
}

Reply:
Can we at least apply the appropriate level of padding just using nesting selectors and changing the `__link--PaddingLeft` value, so that we don't need the `pf-m-nested-level-n` modifiers, or is there value in keeping those?